service/create/service.go

```diff
@@ -384,7 +388,7 @@ func (s *Service) runMachine(input runMachineInput) error {
 	}
 
 	var reservation *ec2.Reservation
-	for i := 0; i < 5; i++ {
+	for i := 0; i < runInstancesRetries; i++ {
```

Comment:
FYI there is something like https://godoc.org/github.com/cenkalti/backoff. Not that critical, but in general I would favour to use some official libs instead of home cooking certain things.